### DISCUSSION TITLE
Update version for Microsoft Edge pls :((

### DISCUSSION THREAD

**Discussion by NguyenASang** [darkreader/darkreader#6864] [Q&A]
It's been 3 days since the release date of version 4.9.35 and now version 4.9.36 is released but the extension on the edge add-ons store has not been updated yet.
Pls update it :((

**Comment by bershanskiy** [comment]
Chrome Web Store review is much faster than Microsoft Store one, so if there are any bugs in release, we can publish fixed version to Chrome Web Store much faster than to Microsoft Store. We hold off Edge publication to keep Edge version bug-free.

Also, if you really need the latest Dark Reader version, you can try installing it form Chrome Web Store. (Not officially supported, but I'll try to fix all reported bugs promptly.)

<details>

<summary>Guide</summary>

![image](https://user-images.githubusercontent.com/45960703/134334592-ad6fecc3-acad-460e-a95d-b6231011dfa5.png)
</details>

**Reply by NguyenASang** [reply]
I know that but if I do that, Is it cause any bug when use on edge ?

**Reply by NguyenASang** [reply]
Maybe I'll change this, thanks for the quick and detailed answer :D Have a nice day
